js/luno.js

```diff
@@ -126,6 +126,34 @@ module.exports = class luno extends Exchange {
         return this.parseOrderBook (orderbook, timestamp, 'bids', 'asks', 'price', 'volume');
     }
 
+    async fetchOrder (id, symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        let response = await this.privateGetOrders (this.extend ({
+            'id': id.toString (),
+        }, params));
+        let state = (response['state'] == 'PENDING') ? 'open' : 'closed';
+        let side = (response['type'] == 'ASK') ? 'sell' : 'buy';
+        return {
+            'id': response.order_id,
+            'datetime': this.iso8601 (response['creation_timestamp']),
+            'timestamp': response['creation_timestamp'],
+            'status': state,
+            'symbol': 'BTC/ZAR', // Assuming because it is not included in response from exchange.
```

Comment:
The transpilation problem is probably because the comments need to be alone in a line. I know, transpilation sucks